Assets/Shopify/UIToolkit/Components/LRUCache.cs

```diff
@@ -0,0 +1,136 @@
+ namespace Shopify.UIToolkit {
```

Comment:
These are nice tools, we might wanna consider moving them to a more general namespace at some point